testsuite/refresh-token-jwt-keycloak-test/src/test/resources/simplelogger.properties

```diff
@@ -0,0 +1 @@
+org.slf4j.simpleLogger.log.io.strimzi=TRACE
```

Comment:
Is this intentional? Ot some debugging left-over?

Reply:
You mean it increases disk consumption and slows the build? I can remove it.

Reply:
No, I meant what I wrote there. I just wondered if it is intentional or some unintentional left-over. It seems a bit unusual to have the TRACE level there by default, that is all. It seems to be tests only - so if it was intentional and you think having it as TRACE by default is what we want just ignore this comment and keep it.

Reply:
I see. Well the next PR that comes after this one is 'testsuite refactoring' and this module is removed. So we can as well leave this for now. But in principle if build speed and disk consumption weren't a consideration I would prefer to have logging set to TRACE all the time. Every additional iteration due to lack of info to understand why the test has failed is additional time spent, and TRACE saves that time.